### PR TITLE
Apply analyzers suggestions

### PR DESCRIPTION
**Pull request by sungam3r**
https://learn.microsoft.com/en-us/dotnet/fundamentals/code-analysis/style-rules/ide0007-ide0008
https://github.com/JosefPihrt/Roslynator/blob/main/docs/analyzers/RCS1118.md
https://github.com/JosefPihrt/Roslynator/blob/main/docs/analyzers/RCS1214.md

### CODE REVIEW

File: src/GraphQL.Tests/BasicQueryTestBase.cs [hunk 1]
```diff
@@ -34,8 +34,8 @@ public ExecutionResult AssertQuery(Action<ExecutionOptions> options, ExecutionRe
     {
         var runResult = Executer.ExecuteAsync(options).Result;
 
-        var writtenResult = Writer.Serialize(runResult);
-        var expectedResult = Writer.Serialize(expectedExecutionResult);
+        string writtenResult = Writer.Serialize(runResult);
```
Comment (Shane32) [inline src/GraphQL.Tests/BasicQueryTestBase.cs:37]: I do not think these suggestions should be implemented

File: src/GraphQL.Tests/Bugs/ArrayOfArrayTest.cs [hunk 2]
```diff
@@ -7,15 +7,15 @@ public class ArrayOfArrayTest : QueryTestBase<ArrayOfArraySchema>
     [Fact]
     public void ArrayOfArray_Should_Return_As_Is()
     {
-        var query = """
+        const string query = """
```
Comment (Shane32) [inline src/GraphQL.Tests/Bugs/ArrayOfArrayTest.cs:10]: This is fine

File: src/GraphQL.Tests/Bugs/BubbleUpTheNullToNextNullable.cs [hunk 3]
```diff
@@ -238,7 +238,7 @@ private void AssertResult(string query, string expected, Data data, IReadOnlyLis
         {
             actualErrors.Count.ShouldBe(errors.Count);
 
-            for (var i = 0; i < errors.Count; i++)
+            for (int i = 0; i < errors.Count; i++)
```
Comment (Shane32) [inline src/GraphQL.Tests/Bugs/BubbleUpTheNullToNextNullable.cs:241]: This also I wouldn’t implement this suggestion

File: src/GraphQL.Tests/Bugs/Bug2194.cs [hunk 4]
```diff
@@ -12,7 +12,7 @@ public void Should_Print_Default_Values_Of_Arguments()
         CultureTestHelper.UseCultures(() =>
         {
             var printer = new SchemaPrinter(new Bug2194Schema(), new SchemaPrinterOptions { IncludeDeprecationReasons = false, IncludeDescriptions = false });
-            var printed = printer.Print();
+            string printed = printer.Print();
```
Comment (Shane32) [inline src/GraphQL.Tests/Bugs/Bug2194.cs:15]: I think we should leave the codebase as is with var being preferred

Comment (sungam3r) [reply src/GraphQL.Tests/Bugs/Bug2194.cs:15]: Editorconfig contains rule that states var should be used for all types except primitive ones - int, string, etc. It has always been so.

File: src/GraphQL.Tests/Bugs/BubbleUpTheNullToNextNullable.cs (hunk 3 above)
Comment (sungam3r) [reply src/GraphQL.Tests/Bugs/BubbleUpTheNullToNextNullable.cs:241]: https://github.com/graphql-dotnet/graphql-dotnet/blob/ac4f9c94e79163036fad38510b2829de4e800f1b/.editorconfig#L87

Comment (sungam3r) [reply src/GraphQL.Tests/Bugs/BubbleUpTheNullToNextNullable.cs:241]: Note, that only test project was changed. Others have already conform this rule.

File: src/GraphQL.Tests/BasicQueryTestBase.cs (hunk 1 above)
Comment (sungam3r) [reply src/GraphQL.Tests/BasicQueryTestBase.cs:37]: https://github.com/graphql-dotnet/graphql-dotnet/blob/ac4f9c94e79163036fad38510b2829de4e800f1b/.editorconfig#L87

**Review by Shane32 — APPROVED** [review]

File: src/GraphQL.Tests/Bugs/BubbleUpTheNullToNextNullable.cs (hunk 3 above)
Comment (Shane32) [reply src/GraphQL.Tests/Bugs/BubbleUpTheNullToNextNullable.cs:241]: Ok

File: src/GraphQL.Tests/Bugs/Bug2194.cs (hunk 4 above)
Comment (Shane32) [reply src/GraphQL.Tests/Bugs/Bug2194.cs:15]: Ok

**Pull request merged**